app/models/post.rb

```diff
@@ -3,7 +3,7 @@ class Post < ActiveRecord::Base
   has_and_belongs_to_many :tags
   has_many :comments, as: :parent
 
-  validates_presence_of :user, :title
+  validates_presence_of :user, :title, :body_markdown
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.